cypress/support/pages/ContactUsPage.js

```diff
@@ -0,0 +1,59 @@
+var emailLocator = 'input[id="email"]';
+var orderReferenceLocator = 'input[id="id_order"]';
+var messageLocator = 'textarea[id="message"]';
+var submitMessageLocator = 'button[id="submitMessage"]';
+var pageURL = 'http://automationpractice.com/index.php?controller=contact';
+var email = 'email@email.com'
+var invalidEmail = 'emailemail.com'
+
+class ContactUsPage {
+  constructor () {}
+
+  	openContactUs () {
+    	cy.visit(pageURL)
+    	cy.url().should('eq', pageURL)
+  	}
+
+  	selectSubjectHeading (destination) {
```

Comment:
Nice to see considering selecting a destination if no destination is passed. then selecting it randomly.